users/replicaJunction/replicaJunction_config.h

```diff
@@ -0,0 +1,75 @@
+#ifndef USERSPACE_CONFIG_H
```

Comment:
Could you replace the first few lines, eg, these lines: 
```c
#ifndef USERSPACE_CONFIG_H
#define USERSPACE_CONFIG_H
```
with just `#pragma one`, and remove the endif at the end? 


Additionally, you should rename this to just `config.h`.  This way, it will be picked up automatically, much like the keymap's config.h and processed.  And then there is no need to actually include this file on your keymap's config.